src/components/Widgets/Widget.tsx

```diff
@@ -7,52 +8,34 @@ import { useActiveTabStore } from '@/stores/activeTab/ActiveTabStore';
 import { useMenuStore } from '@/stores/menu';
 import { useSettingsStore } from '@/stores/settings';
 import type { LanguageKey } from '@/types/i18n';
-import type { StarterVariantType, ThemeVariantType } from '@/types/internal';
 import type { MenuState } from '@/types/menu';
-import { ChainId, EVM } from '@lifi/sdk';
+import { EVM } from '@lifi/sdk';
 import type { WidgetConfig } from '@lifi/widget';
 import { HiddenUI, LiFiWidget } from '@lifi/widget';
 import { useTheme } from '@mui/material/styles';
 import { getWalletClient, switchChain } from '@wagmi/core';
-import { useMemo, useState } from 'react';
+import { useMemo } from 'react';
 import { useTranslation } from 'react-i18next';
+import { ThemesMap } from 'src/const/themesMap';
+import { useMemelist } from 'src/hooks/useMemelist';
 import { darkTheme } from 'src/theme/theme';
 import { useConfig } from 'wagmi';
 import { WidgetWrapper } from '.';
-import { useMemelist } from 'src/hooks/useMemelist';
-import { ThemesMap } from 'src/const/themesMap';
+import type { WidgetProps } from './Widget.types';
+import { refuelAllowChains, themeAllowChains } from './Widget.types';
 import { WidgetSkeleton } from './WidgetSkeleton';
-import { ClientOnly } from '@/components/ClientOnly';
-
-const refuelAllowChains: ChainId[] = [
-  ChainId.ETH,
-  ChainId.POL,
-  ChainId.BSC,
-  ChainId.DAI,
-  ChainId.FTM,
-  ChainId.AVA,
-  ChainId.ARB,
-  ChainId.OPT,
-  ChainId.BAS,
-  ChainId.MAM,
-];
-
-const themeAllowChains: ChainId[] = [
-  ChainId.ETH,
-  ChainId.BAS,
-  ChainId.OPT,
-  ChainId.ARB,
-  ChainId.AVA,
-  ChainId.BSC,
-];
-
-interface WidgetProps {
-  starterVariant: StarterVariantType;
-  themeVariant?: ThemeVariantType;
-}
 
-export function Widget({ starterVariant, themeVariant }: WidgetProps) {
-  const [loaded, setLoaded] = useState(false);
+export function Widget({
+  starterVariant,
+  fromChain,
```

Comment:
Might block the current widget implementation if we do not have default value for those

Reply:
Valid point, but it does not seem to break anything.

<img src="https://media4.giphy.com/media/dngVpFeyJ7nSU/giphy.gif"/>